src/plugins/discord/sqliteMirror.test.js

```diff
@@ -256,6 +258,37 @@ describe("plugins/discord/sqliteMirror", () => {
       });
     });
 
+    describe("nthMessageIdFromTail", () => {
+      function buildMessages() {
+        const channelId: Snowflake = "1";
+        const authorId: Snowflake = "2";
+
+        const db = new Database(":memory:");
+        const sqliteMirror = new SqliteMirror(db, "0");
+        sqliteMirror.addUser(testUser(authorId));
+        sqliteMirror.addChannel(testChannel(channelId));
+
+        const maxCount = 10;
+        var x = 1;
+        while (x <= maxCount) {
```

Comment:
This would be cleaner as a for-loop I think :]
```js
for(let x = 1; x <= 10; x++)
```

But when you do use mutable variables, prefer using `let` over `var`.

Reply:
updated